### DISCUSSION TITLE
AYO what the fuck just happened?

### DISCUSSION THREAD

**Discussion by KitOdin** [FunkinCrew/Funkin#1691] [Q&A]
i was busy staring at gf but slightly hue-shifted when suddenly the intro of the game play but it was more high quality than expected. explanation plz?

**Comment by isakube** [comment]
you pressed an arrow key on the newgrounds version :trolle:

**Reply by KitOdin** [reply]
no it appears it triggered after a few minutes of idling on the title screen.

**Reply by s-zenmode** [reply]
> no it appears it triggered after a few minutes of idling on the title screen.

known bug.
https://github.com/ninjamuffin99/Funkin/issues/1136

**Comment by NokioPyro** [comment]
Maybe it could be a bug or ninjamuffin made it so after a certain amount of time it would make gf change hue just like with the arrow keys, but i dont recall there being a cur beat or cur step in title state, as its not in song,
